front/SettingsForm.tsx

```diff
@@ -81,6 +82,7 @@ const SettingsForm = () => {
               className="form-control form-control-lg"
               rows={8}
               placeholder="Short bio about you"
+              data-cy="settings-bio"
```

Comment:
The data-cy attribute value for the bio textarea should be 'settings-bio-input' for better clarity.